easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.4.2-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '3.4.2'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://ffmpeg.org/releases/']
+
+dependencies = [
+    ('NASM', '2.13.02'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('x264', '20180128'),
+    ('LAME', '3.100'),
+    ('x265', '2.6'),
+    ('X11', '20180131'),
+]
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore toolchain
+    ('binutils', '2.28'),
+]
+
+configopts = '--enable-pic --enable-shared --enable-gpl --enable-version3 --enable-nonfree --cc="$CC" --cxx="$CXX" '
+configopts += '--enable-libx264'
```

Comment:
@RvDijk Any reason you removed the `--enable-libx265 --enable-libmp3lame` we have in existing easyconfigs?

Reply:
@boegel nope, probably took an older version as a template...